DataScience/ExperimentationAzure.py

```diff
@@ -3,7 +3,7 @@
 from subprocess import check_output, STDOUT
 from AzureUtil import AzureUtil
 import Experimentation
-import ImportantFeatures
+import FeatureImportance
```

Comment:
It should be FeaturesImportance since there is usually more than 1 feature

Reply:
Wont fix after discussing with Marco.

---
In reply to: [283940769](https://github.com/microsoft/mwt-ds/pull/132#discussion_r283940769) [](ancestors = 283940769)